### ISSUE TITLE
[Feature Request] Oxen wallet lost dark theme after Loki renaming

### ISSUE THREAD

**Issue by invalid-error** [loki-project/loki-electron-gui-wallet#251]
Please bring back a dark, grey or dark blue (night) theme.

**Comment by argonatorUberTF** [comment]
Completely possible to do and add this feature. Will check electron source from other wallets and propose a PR.

**Comment by mdPlusPlus** [comment]
I agree, I saw the old theme the first time today and would love to be able to use it as a generic dark mode option.
